apps/web/types.ts

```diff
@@ -0,0 +1,32 @@
+export interface CourseCardProps {
+	courseName: string;
+	courseCode: string;
+	coursePrerequisites?: string[];
+	courseRating: number;
+	courseReviewCount: number;
+	description: string;
+}
+
+export interface MyReviewsData {
+	academicYear: number;
+	course: {
+		code: string;
+		id: number;
+	};
+	created_at: string;
+	description: string;
+	id: number;
+	isOwner: boolean;
+	profile: {
+		id: string;
+	};
+	rating: number;
+	status: string;
+	votes: number;
+}
```

Comment:
- Can you replace stuff inside `CourseCardProps` with [this course type?](https://github.com/stamford-syntax-club/course-compose/blob/main/apps/web/types/course.ts). Also, I think props type can be put back inside `course-card.tsx` unless we plan to use it somewhere else?
```ts
export interface CourseCardProps {
   course Course
}
```

- `MyReviewsData` is actually the same thing as a normal review response like [this one ](https://github.com/stamford-syntax-club/course-compose/blob/main/apps/web/types/reviews.ts). Additionally, since the backend returns in a paginated format, I recommend you use it with [PaginatedResponse](https://github.com/stamford-syntax-club/course-compose/blob/main/apps/web/types/pagination.ts) like `PaginatedResponse<Review>` and you will have access to extra stuff like pageNumber, pageSize, totalNumberOfItems, totalPages.

- I have created a folder called `types`, can you put this code over there? Perhaps separate each type in separated files. Thank you :3